packages/components/organisms/f-cookie-banner/test/specs/component/shared/f-cookieConsentBanner.component.spec.js

```diff
@@ -26,9 +31,34 @@ describe('New - f-cookieBanner component tests', () => {
 
 describe('New - Multi-tenant - f-cookieBanner component tests', () => {
     forEach(['es', 'ie', 'it']) // 'dk' and 'no' disabled for now
-    .it.only('should display the f-cookieBanner component', (tenant) => {
+    .it.only('should display the f-cookieBanner component', tenant => {
```

Comment:
Remove `.only`

You should make the `tenant` param the full tenant code, then we can remove the switch 👍🏼